led_strip/src/led_strip_spi_dev.c

```diff
@@ -0,0 +1,212 @@
+/*
+ * SPDX-FileCopyrightText: 2022-2023 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <stdlib.h>
+#include <string.h>
+#include <sys/cdefs.h>
+#include "esp_log.h"
+#include "esp_check.h"
+#include "driver/spi_master.h"
+#include "driver/spi_common.h"
+#include "led_strip.h"
+#include "led_strip_interface.h"
+#include "hal/spi_hal.h"
+
+#define LED_STRIP_SPI_DEFAULT_RESOLUTION (2.4 * 1000 * 1000) // 10MHz resolution
```

Comment:
wrong comment?

Reply:
and please note, because the SPI hardware doesn't have the fractional divisor, so this might not be the real resolution. Please call spi_device_get_actual_freq to get the final clock frequency.